redhat/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/rhel
```

Comment:
Is there a way to make the dependency on RHEL 7 more explicit/locked? `/rhel:7` maybe? Not sure which tags are available.

Reply:
Fixed.